coalib/results/result_actions/ResultAction.py

```diff
@@ -12,6 +12,9 @@ class ResultAction:
 
     SUCCESS_MESSAGE = 'The action was executed successfully.'
 
+    def __init__(self):
+        self.next_actions = []
+
     @staticmethod
     def is_applicable(result, original_file_dict, file_diff_dict):
```

Comment:
hmm, we can check the applicability of the ResultAction at the time of instantiation, but then it'd need to be initialized only after the Result is available.